packages/catalog-process/test/integration/getEserviceById.test.ts

```diff
@@ -81,7 +81,7 @@ describe("get eservice by id", () => {
       eservice1.id,
       getMockContext({ authData })
     );
-    expect(result).toStrictEqual(eservice1);
+    expect(result.data).toStrictEqual(eservice1);
```

Comment:
As done in other services, when we added the metadata to the return of the service, let's always test the entire return object

```suggestion
    expect(result).toStrictEqual({
      data: eservice1,
      metadata: { version: 0 },
    });
```